src/app/globals.css

```diff
@@ -72,3 +72,25 @@ h3 {
   font-weight: 300;
   text-align: center;
 }
+
+.article {
+  background-color: var(--card-bg);
+  border-radius: var(--border-radius);
+  box-shadow: 0 7px 12px -7px rgba(20, 83, 118, 0.39);
+}
+
+.page {
+  height: 100%;
+  background: rgb(201, 212, 242);
+  background: linear-gradient(
+    150deg,
+    rgba(201, 212, 242, 1) 3%,
+    rgba(228, 247, 255, 1) 30%
+  );
+}
+
+.heading {
+  font-family: var(--card-heading-font-family);
+  font-size: var(--card-heading-font-size);
+  color: var(--text-color-primary);
+}
```

Comment:
This should not be added here. Please double-check where it is used and move it to it's CSS file

Reply:
Done. Thank you!